src/compiler/declarationEmitter.ts

```diff
@@ -399,7 +399,7 @@ module ts {
 
             function emitTypeQuery(type: TypeQueryNode) {
                 write("typeof ");
-                emitEntityName(type.exprName);
+                emitEntityName(<any>type);

```

Comment:
No. :)

Also, exprName may not be an entity name.  So this could just die in that call.  You'll need to actually handle emitting this properly given that you could have: this/super/property-access, etc.